app.js

```diff
@@ -58,6 +58,21 @@ app.use((err, req, res, _next) => {
   res.status(err.status || 500).send(errorMessage);
 });
 
-app.listen(config.httpPort, () => {
+const server = app.listen(config.httpPort, () => {
   console.log(`HTTP server listening on port ${config.httpPort}.`);
 });
+
+
+process.on('SIGINT', () => {
```

Comment:
how about doing the same for SIGTERM as well ? SIGTERM is what happens when we "ctrl-C" in the terminal. SIGINT is what pm2 sends when we call stop.